projects/ion/src/lib/tab/tab.component.spec.ts

```diff
@@ -57,4 +56,10 @@ describe('TabComponent', () => {
     await sut({ label: defaultName, disabled: true });
     expect(screen.getByText(defaultName)).toBeDisabled();
   });
+
+  it('should render icon on tab', async () => {
+    await sut({ label: defaultName, iconType: 'trash' });
+    const elementRendered = document.getElementById('ion-icon-trash');
```

Comment:
You can improve this returning `document.getElementById('ion-icon-trash');` in sut.